src/OpenTelemetry.SemanticConventions/AspnetcoreAttributes.cs

```diff
@@ -0,0 +1,119 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+// <auto-generated>This file has been auto generated from scripts/templates/SemanticConventionsAttributes.cs.j2</auto-generated>
+
+#pragma warning disable CS1570 // XML comment has badly formed XML
+
+using System;
+
+namespace OpenTelemetry.SemanticConventions;
+
+/// <summary>
+/// Constants for semantic attribute names outlined by the OpenTelemetry specifications.
+/// </summary>
+public static class AspnetcoreAttributes
```

Comment:
I think it's worth adding a special case to rename this class to `AspNetCoreAttributes`. Same for the constants.